src/lib/NonPartitionedSessionStorage.js

```diff
@@ -182,6 +216,14 @@ class NonPartitionedSessionStorage {
 NonPartitionedSessionStorage.COOKIE_MAX_AGE = 7 * 60; // 7 minutes; in seconds
 NonPartitionedSessionStorage.TOP_LEVEL_SESSION_STORAGE_ENTRY = '_topLevelEntry';
 NonPartitionedSessionStorage.IFRAME_SESSION_STORAGE_ENTRY = '_iframeEntry';
+/**
+ * @readonly
+ * @enum { '1' | '0' }
+ */
+NonPartitionedSessionStorage.PREFIX_FLAG_SESSION_STORAGE_PARTITIONED = {
```

Comment:
The word session storage is twice in this. Suggestion to rename:
```suggestion
NonPartitionedSessionStorage.PREFIX_FLAG_IS_PARTITIONED = {
```

Reply:
The name `PREFIX_FLAG_SESSION_STORAGE_PARTITIONED` is meant as "prefix" of the "FLAG_SESSION_STORAGE_PARTITIONED" cookie.